### PR TITLE
Containers

### PR DESCRIPTION
**Pull request by MonsterFich**
Added "Container","Nodes".Modified "Program.cs".

### CODE REVIEW

File: XMLDocGen/Tree/Container.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace XMLDocGen
+{
+    public class Container
+    {
+        protected List<XMLDocGen.Models.Member> Members { get; }
+
+        //protected List<Tree.Models.INode> Nodes { get; set; } = new List<Tree.Models.INode>();
```
Comment (nsubaru) [inline XMLDocGen/Tree/Container.cs]: Remove commented code

File: XMLDocGen/Tree/Models/NodeFunction.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        public string Name { get; }
+
+        public string Description { get; }
+
+        public UInt16 Args { get; }
+
+        public string Returning { get; }
+
+        public NodeFunction(string name, string desc, UInt16 args, string ret)
```
Comment (nsubaru) [inline XMLDocGen/Tree/Models/NodeFunction.cs]: args is count of function\method arguments or it's your mistake and it's must be list of arguments

File: XMLDocGen/Tree/Models/NodeMethod.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+        public string Name { get; }
+
+        public string Description { get; }
+
+        public UInt16 Args { get; }
+
+        public string Returning { get; }
+        
+        public NodeMethod(string name, string desc, UInt16 args,string ret)
```
Comment (nsubaru) [inline XMLDocGen/Tree/Models/NodeMethod.cs]: Same as previous

**Review by nsubaru — CHANGES_REQUESTED** [review]
Fix all what required in ccomments

File: XMLDocGen/Tree/Models/NodeFunction.cs (hunk 2 above)
Comment (nsubaru) [reply XMLDocGen/Tree/Models/NodeFunction.cs]: See https://github.com/Nelyndir/XDCtoMarkdown/blob/a862176101bf7f2ed90a7ca0d37ab825994ac9f2/XMLDocGen/Models/Method.cs#L22

**Review by nsubaru — APPROVED** [review]

**Review by Nelyndir — APPROVED** [review]

**Pull request merged**